Knative Social Media Guidelines.md

```diff
@@ -0,0 +1,62 @@
+# Knative Social Media Guidelines
+
+This document covers the social media guidelines for the use of the Knative Twitter/X channel.
+
+**Knative Project Twitter/X handle:**
+- [https://twitter.com/KnativeProject](https://twitter.com/KnativeProject)
+- @KnativeProject
+
+**LinkedIn company page:**
+- [https://www.linkedin.com/company/knative/](https://www.linkedin.com/company/knative/)
+
+## Objectives
+
+The objectives of project social media are:
+
+- Share project news, roadmap updates, news releases.
+- Report on performance and security updates.
+- Educate on project updates.
+- Share information on how-tos, demos, etc. that are valuable to the community.
+- Build awareness for the project.
+- Drive engagement and participate in the project.
+
+## Strategy and Guidelines
+
+### Strategy
+
+- Keep messages positive and uplifting, consistent with the values and principles of CNCF.
+- Communicate big picture ideas vs “announcements” or news. News will be positioned less like an announcement and more around what it means.
+- Share content across social media channels that benefit the ecosystem as a whole.
+- Maintain a balance of posting on CNCF activities/news and ecosystem-focused content.
+- Share vendor-neutral, community-sourced posts that are informational, engaging, and ecosystem-focused.
+- Engage with the community through retweets and sharing of community content.
+
+### General Guidelines
```

Comment:
Would you want to mention anything around the frequency? I am not an expert in social media, but we wouldn't want to spam followers with very frequent posts. Similarly, we wouldn't want to post too infrequently.

In some groups, I've seen that they decide on a social media post every community meeting. Perhaps we can introduce something like that...

- In every Knative SC meeting (happens every 2 weeks), the SC can decide on what to post next week and the week after.
- With folks assigned to the tasks of creating a blurb for the post.

Reply:
It feels like an ideal cadence might be weekly or twice-weekly, but I'm not sure we're up for producing content that fast yet.